init.lua

```diff
@@ -3,6 +3,19 @@ ts_furniture = {}
 -- If true, you can sit on chairs and benches, when right-click them.
 ts_furniture.enable_sitting = true
 
+local valid_player_model_versions =  {
+	default_character_v1 = true,
+	default_character_v2 = true,
+	default_character_v3 = true,
+}
+local player_model_version = minetest.settings:get("player_model_version")
```

Comment:
You could simply check whether the mod `player_api` exists. Yes: new model, No: old model.